dummy-api/matrix-service-api.yaml

```diff
@@ -0,0 +1,115 @@
+swagger: '2.0'
+info:
+  title: HCA Matrix Service
+  version: 0.0.0-dummy
+schemes:
+  - http
+  - https
+consumes:
+  - application/json
+produces:
+  - application/json
+securityDefinitions:
+  api_key:
+    type: apiKey
+    name: x-api-key
+    in: header
+definitions:
+  MatrixRequest:
+    type: object
+    properties:
+      bundle_fqids:
+        type: array
+        items:
+          type: string
+          format: dcp-fqid
+      bundle_fqids_url:
+        type: string
+        format: url
+      filter:
+        $ref: '#/definitions/MatrixFilter'
+      format:
+        $ref: '#/definitions/MatrixFormat'
+  MatrixFilter:
+    type: object
+    properties:
+      pandas_filter_string:
+        type: string
+  MatrixFormat:
+    type: string
+    enum:
+      - zarr
+      - parquet
+      - loom
+      - anndata
+      - singlecellexperiment
+  MatrixResponse:
+    type: object
+    properties:
+      request_id:
+        type: string
+        format: uuid
+      status:
+        $ref: '#/definitions/MatrixRequestStatus'
+      key:
+        type: string
+  MatrixRequestStatus:
```

Comment:
[Azure Best Practices](https://docs.microsoft.com/en-us/azure/architecture/best-practices/api-design#asynchronous-operations)

_If the client sends a GET request to this endpoint, the response should contain the current status of the request. **Optionally, it could also include an estimated time to completion or a link to cancel the operation**_

`{`
   ` "status":"In progress",`
    `"link": { "rel":"cancel", "method":"delete", "href":"/api/status/12345" }`
`}`

Reply:
So I can add a placeholder field for estimated time or some other value that represents the scale of the request. I'm a bit concerned that the orange box will depend on it prematurely though. It may not convey useful information in our v0 release.